lib/fastlane/plugin/android_sdk_update/actions/android_sdk_update_action.rb

```diff
@@ -52,6 +68,15 @@ def self.run(params)
         end
       end
 
+      def self.download_and_extract_sdk(download_url, sdk_path)
+        FastlaneCore::CommandExecutor.execute(
+          command: "wget -O /tmp/android-sdk-tools.zip #{download_url}",
+          print_all: true, print_command: true)
+        FastlaneCore::CommandExecutor.execute(
+          command: "unzip /tmp/android-sdk-tools.zip -d #{sdk_path}",
```

Comment:
We should use the silent option on unzip, because it produces a lot of output

Reply:
Good call. Done.